packages/playground/blueprints/src/lib/steps/run-sql.ts

```diff
@@ -0,0 +1,90 @@
+import { StepHandler } from '.';
+import { rm } from './rm';
+
+/**
+ * @inheritDoc runSql
+ * @hasRunnableExample
+ * @example
+ *
+ * <code>
+ * {
+ *		"step": "runSql",
+ *		"sql": {
+ *			"resource": "literal",
+ *			"name": "schema.sql",
+ *			"contents": "SELECT * FROM wp_posts"
```

Comment:
SELECT effectively won't do anything here, let's use an example where the effects can be observed:

```suggestion
 *			"contents": "DELETE FROM wp_posts"
```

Reply:
Done: https://github.com/WordPress/wordpress-playground/pull/745/files#diff-ef7fc55c9e2c228ab627cd90d8372d926a0d8625f5f8fc76baf10cd15ed4f85fR16